blazor/diagram/user-handle.md

```diff
@@ -705,6 +787,84 @@ The following code explains how to customize the connector fixed user handle.
+
+
+```csharp
+@using Syncfusion.Blazor.Diagram
+
+<SfDiagramComponent @ref="@diagram" Height="600px" Nodes = "@nodes" Connectors="@connectors">
+    <DiagramTemplates>
+        <FixedUserHandleTemplate>
+            if ((context as FixedUserHandle).ID == "user1" || (context as FixedUserHandle).ID == "user2")
+            {
+                <div id="button" style="height: 100%; width: 100%;">
+                    <input type="button" value="Button1" />
+                </div>
+            }
+        </FixedUserHandleTemplate>
+    </DiagramTemplates>
+</SfDiagramComponent>
+
+@code
+{
+    public SfDiagramComponent diagram;
+
+    DiagramObjectCollection<Node> nodes = new DiagramObjectCollection<Node>();
+    DiagramObjectCollection<Connector> connectors = new DiagramObjectCollection<Connector>();
+
+    protected override void OnInitialized()
+    {
+        Node node1 = new Node()
+        {
+            ID="node1",
+            OffsetX = 250,
+            OffsetY = 250,
+            Width = 100,
+            Height = 100,
+            Style = new ShapeStyle() { Fill = "#6495ED", StrokeColor = "white" },
+            FixedUserHandles = new DiagramObjectCollection<NodeFixedUserHandle>()
+        {
+            new NodeFixedUserHandle()
+            {
+                ID = "user1",
+                    Height = 30,
+                    Width = 30,
+                    Visibility = true,
+                    Padding = new DiagramThickness() { Bottom = 1, Left = 1, Right = 1, Top = 1 },
+                    Margin = new DiagramThickness() { Left = 30 }, Offset = new DiagramPoint() { X = 1 , Y = 0 }
+                },
+             }
+            };
+        nodes.Add(node1);
+
+        Connector connector = new Connector
+        {
+            ID = "connector1",
+            Type = ConnectorSegmentType.Orthogonal,
+            SourcePoint = new DiagramPoint() { X = 600, Y = 200 },
+            TargetPoint = new DiagramPoint() { X = 800, Y = 400 },
+            FixedUserHandles = new DiagramObjectCollection<ConnectorFixedUserHandle>() { new ConnectorFixedUserHandle() { 
+                ID="user2",
+                Offset = 0.5, 
+                Alignment = FixedUserHandleAlignment.Center, 
+                Displacement = new DiagramPoint() { X = 10}, 
+                Padding = new DiagramThickness() { Bottom = 1, Left = 1, Right = 1, Top = 1 }, 
+                Width = 30, 
+                Height = 30, 
+            }
+            }
+        };
+        connectors.Add(connector);
+    }
+}
+```
+You can download a complete working sample from [GitHub](https://github.com/SyncfusionExamples/Blazor-Diagram-Examples/tree/master/UG-Samples/UserHandle)
```

Comment:
You can download a complete working sample from [GitHub](https://github.com/SyncfusionExamples/Blazor-Diagram-Examples/tree/master/UG-Samples/UserHandle).